### ISSUE TITLE
Traderep Report should show you unrepped trades

### ISSUE THREAD

**Issue by AardWolf** [AardWolf/aardcogs#111]
Just add a line to the report to show which completed trades you haven't repped and who they were with.

**Comment by AardWolf** [comment]
```
You have None rep over 0 repped trades.
You are a good trading partner. Or at least you have no unrepped completed trades.
You have good trading partners or at least none you are waiting for rep from.
```
```
You have 2 rep over 2 repped trades.
Completed trades waiting for you to rep them: trade 10 with ad, trade 11 with ad, trade 12 with ad
Partners you might want to poke so they rep you: trade 2 with ad, trade 9 with ad, trade 10 with ad, trade 11 with ad, trade 12 with ad
```

Perhaps the formatting needs some work. It shouldn't say "None".
Probably just list the people with trade num in parens. Or provide a usage string example?

**Comment by AardWolf** [comment]
Fixed in #115
